config/locales/en/en.yml

```diff
@@ -124,13 +133,6 @@ en:
         accent_main_color: "Color of links, buttons, etc. Defaults to banner BG color."
         accent_fg_color: "Color of text on top of accent color. Defaults to banner FG color."
     labels:
-      project_step:
-        actual_end_date: "Completed Date"
-        duration: "Scheduled Duration"
-        is_finalized: "Finalized"
-        scheduled_duration_days: "Scheduled Duration"
-        scheduled_start_date: "Scheduled Start Date"
-        step_type_value: "Type"
       timeline_step_move:
```

Comment:
I wasn't able to move `timeline_step_move` since it is not an active record model.